src/components/Nav/Nav.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {BrowserRouter as Router} from 'react-router-dom';
+import Nav from './Nav';
+
+describe('Nav', () => {
+  it('renders without crashing', () => {
+    const div = document.createElement('div');
+    ReactDOM.render(
+      <Router>
+        <Nav />
+      </Router>,
+      div
+    );
+    ReactDOM.unmountComponentAtNode(div);
```

Comment:
If it does crash then it won't unmount the component. 

Can you add `enzyme` to this repo and use that instead plz.